packages/jest-cli/src/snapshot_interactive_mode.js

```diff
@@ -13,68 +13,163 @@ import type {AggregatedResult} from 'types/TestResult';
 const chalk = require('chalk');
 const ansiEscapes = require('ansi-escapes');
 const {pluralize} = require('./reporters/utils');
-const {KEYS} = require('./constants');
+const {KEYS, ARROW} = require('./constants');
 
 export default class SnapshotInteractiveMode {
   _pipe: stream$Writable | tty$WriteStream;
   _isActive: boolean;
-  _updateTestRunnerConfig: (path: string, shouldUpdateSnapshot: boolean) => *;
-  _testFilePaths: Array<string>;
+  _updateTestRunnerConfig: (
+    assertion: ?Object,
+    shouldUpdateSnapshot: boolean,
+  ) => *;
+  _testAssertions: Array<Object>;
   _countPaths: number;
+  _skippedNum: number;
 
   constructor(pipe: stream$Writable | tty$WriteStream) {
     this._pipe = pipe;
     this._isActive = false;
+    this._skippedNum = 0;
   }
 
   isActive() {
     return this._isActive;
   }
 
-  _drawUIOverlay() {
+  getSkippedNum() {
+    return this._skippedNum;
+  }
+
+  _clearTestSummary() {
```

Comment:
seems like a useful thing for all watch plugins. fine enough to keep in here for now though, thanks for clarifying!